src/Tools/IdeCoreBenchmarks/IdeCoreBenchmarks.csproj

```diff
@@ -1,21 +1,25 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
 <Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepositoryEngineeringDir)targets\GenerateCompilerExecutableBindingRedirects.targets" />
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
```

Comment:
```suggestion
    <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
```

Reply:
I wouldn't hardcode the netcoreapp3.1 TFM, I think we have a property somewhere that we use for tests that you could use here

Reply:
Thanks. Using `RoslynPortableTargetFrameworks` instead.